src/ClassDesignChallenge/Courses.java

```diff
@@ -0,0 +1,18 @@
+package ClassDesignChallenge;
+
+public class Courses {
```

Comment:
We want to have a class so we would be able to create one single course, or 1000. 
Please consider the fact that in most cases, we don't use the plural.
When we create a course object, it's just one, not a list of them.
This also applies to Professors and Students.

Reply:
Yes indeed.  It also sounds strange having an object at plural. The names of the classes have been refactored.